### PR TITLE
Make Negated field editable with dropdown menu

### PR DESCRIPTION
**Pull request by kthorat-prog**

### CODE REVIEW

File: src/main/cliapp/src/components/datatables/DiseaseAnnotationsComponent.js [hunk 1]
```diff
@@ -43,7 +44,10 @@ export const DiseaseAnnotationsComponent = () => {
     const toast_topleft = useRef(null);
     const toast_topright = useRef(null);
 
-
+    const negatedTerms = [
+        { name: 'True' },
+        { name: 'False' }
+    ];
```
Comment (oblodgett) [inline src/main/cliapp/src/components/datatables/DiseaseAnnotationsComponent.js:50]: These terms should come from the `ControlledVocabularyService` just setup a list in there and give it a name.

Comment (kthorat-prog) [reply src/main/cliapp/src/components/datatables/DiseaseAnnotationsComponent.js:50]: exactly 
exactly like this one - useQuery(['diseaseRelationTerms'],
        () => controlledVocabularyService.getTerms('disease_relation_terms'), {
            onSuccess: (data) => {
                setDiseaseRelationTerms(data)
            }
        }
    )

Comment (oblodgett) [reply src/main/cliapp/src/components/datatables/DiseaseAnnotationsComponent.js:50]: yes but the name is: `generic_boolean_terms`

Comment (kthorat-prog) [reply src/main/cliapp/src/components/datatables/DiseaseAnnotationsComponent.js:50]: yea..got it...thanks

**Pull request closed without merging**